### PR TITLE
watch: update German translation

### PR DESCRIPTION
**Pull request by KRyTeX1337**
Added the watch -g command to the german page

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/watch.md [hunk 1]
```diff
@@ -14,3 +14,7 @@
 - Überwache "node"-Prozesse und aktualisiere alle 3 Sekunden:
 
 `watch -n {{3}} "{{ps aux | grep node}}"`
+
+- Überwache, ob sich der Festplatten-Speicherplatz ändert und höre mit der Überwachung auf, sobald eine Änderung auftritt:
```
Comment (kbdharun) [inline pages.de/linux/watch.md:18]: Thanks for your contribution. Can you add this to the [English page](https://github.com/tldr-pages/tldr/blob/main/pages/linux/watch.md) too?

Comment (KRyTeX1337) [reply pages.de/linux/watch.md:18]: Sure thing :)

Comment (kbdharun) [reply pages.de/linux/watch.md:18]: Merged it, thanks.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]
It's a bit verbose, but very fine

**Pull request merged**